### ISSUE TITLE
JAWS reads contents of disabled button in virtual cursor mode

### ISSUE THREAD

**Issue by ananthula** [FreedomScientific/VFO-standards-support#722]
### Summary

When using tab navigation JAWS does not tab on the disabled button but using virtual cursor, JAWS reads out contents of disabled button
> Example:
> 1. Go to https://codepen.io/robertwbradford/pen/NaMNJg
> 2. in the console output use tab navigation and notice that disabled buttons are not focused.
> 3. Switch on virtual cursor using Insert + Z and use arrow keys, observe that the contents of disabled buttons are nor read.

### Expected result

the contents of disabled button should not be read

### Actual result

> What actually did happen?
the contents of disabled button should are read

### Example

[> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...
](https://codepen.io/robertwbradford/pen/NaMNJg)

#### JAWS version and build number
JAWS 2022.2110.60

 
#### Operating System and version

Windows 10 22H2

#### Browser and version:
Chrome

**Comment by JAWS-test** [comment]
This is not a bug, but correct. When reading with the virtual cursor, all content should be perceivable - this is desired behavior and is also done this way by other screen readers.

Please close the issue.

**Comment by stevefaulkner** [comment]
thanks @JAWS-test, closing issue
